brave/brave-core
# Add active wallet count P3A metrics

Resolves brave/brave-browser#25504

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0. Start browser with fresh profile and `--p3a-count-wallet-test-networks` command line switch.
1. Ensure metrics do not exist on local state page.
2. Create new wallet, create various accounts for Ethereum, Solana and Filecoin. If on Android, close and re-open wallet, wait 10 seconds.
3. Ensure metrics do not exist on local state page.
4. Deposit various testnet native coins and chain tokens (such as ERC20, ERC721 and SPL tokens) into the accounts. Acquire coins/tokens via a testnet faucet.
5. Reload the wallet page, unlock if necessary. If on Android, wait 10 seconds.
6. Check local state page, ensure metrics contain expected values.
7. Start browser without `--p3a-count-wallet-test-networks` switch and open/unlock wallet. If on Android, wait 10 seconds.
8. Ensure metrics contain a zero value.
